docs/JestObjectAPI.md

```diff
@@ -158,7 +158,7 @@ as well as all the modules that it imports._
 
 Modules that are mocked with `jest.mock` are mocked only for the file that calls
 `jest.mock`. Another file that imports the module will get the original
-implementation even if run after the test file that mocks the module.
+implementation even if runs after the test file that mocks the module.
```

Comment:
Hey @shaneu, thanks for the fix

This seems wrong to me, did you mean to change this to:

> even if **it** runs after the test

Reply:
Oh man, I saw that run was missing an 's', totally missed that it was also missing 'it'! Yes, that's what I meant/